d2core/d2map/map_entity.go

```diff
@@ -75,40 +75,48 @@ func (m *mapEntity) Step(tickTime float64) {
 	}
 
 	stepX, stepY := m.getStepLength(tickTime)
+	looped := false
+	for {
+		looped = looped
```

Comment:
🤨

Reply:
I'm fixing these and re-pushing, this was for debugging. Good catch!